activerecord/test/cases/relation/or_test.rb

```diff
@@ -180,14 +181,14 @@ def test_structurally_incompatible_values
   # https://www.sqlite.org/limits.html#max_expr_depth
   class TooManyOrTest < ActiveRecord::TestCase
     unless current_adapter?(:SQLite3Adapter)
-      fixtures :citations
+      fixtures :paragraphs
 
       def test_too_many_or
-        citations = 6000.times.map do |i|
-          Citation.where(id: i, book2_id: i * i)
+        paragraphs = 1001.times.map do |i|
```

Comment:
Is 1001 entries enough to reliably blow the stack here? cc @kamipo

Reply:
It raised for 1000 when the fix was implemented - https://github.com/rails/rails/commit/7d3bff1fdb79dd2c71017804a3f396dc7fe2c9da